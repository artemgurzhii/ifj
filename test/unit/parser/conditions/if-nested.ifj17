if foo == 0 then
  if bar == 1 then
    return 1
  end if
end if
