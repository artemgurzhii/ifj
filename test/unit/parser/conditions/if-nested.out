(if (== (id foo) (int 0))
  (if (== (id bar) (int 1))
    (return
      (int 1))))
